tox.ini

```diff
@@ -128,7 +128,8 @@ commands_pre =
     pip freeze
 commands=
     pytest astropy/astropy/io/misc/asdf --open-files --run-slow --remote-data \
-        -W "ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.types"
+        -W "ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.types" \
+        -W "ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.extension"
```

Comment:
Where in astropy are these failing?

Reply:
https://github.com/asdf-format/asdf/pull/1409#discussion_r1114429115